### PR TITLE
Add Sonarqube docker tag support to resolve #13463

### LINKED ISSUE #13463: SonarQube WebApp + PostgresSql goes into maintenance mode
This is already the second time this happened and I got no idea how to resolve this. I have deployed the following template

[SonarQube on Web App with PostgreSQL and VNet integration](https://github.com/Azure/azure-quickstart-templates/tree/master/quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet)

And after some months my SonarQube installation goes into maintenance mode, by what I guess is by attempting to do an upgrade. The only way I then can fix this is by doing a new deployment and starting with a clean database. Any idea how to do a proper upgrade or how to make sure SonarQube doesn't do an upgrade?

### PR DESCRIPTION
**Pull request by erudinsky**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add Sonarqube docker tag support to resolve #13463
* Change Api versions to recent
* Add section in readme on sonarqube's docker tag

### CODE REVIEW

**Comment by erudinsky** [comment]
@bmoore-msft would you mind to take a look (as you did initial review of this template), this PR addresses couple of issues (see changelog).

File: quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     },
     "administratorLoginPassword": {
       "value": "GEN-PASSWORD"
+    },
+    "sonarqubeDockerTag": {
+      "value": "GEN-SONARQUBE-IMAGE-TAG"
```
Comment (alex-frankel) [inline quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json:15]: I have not seen this image tag before, can you confirm that this deployment not only succeeded, but the app is running as expected?

**Review by alex-frankel — CHANGES_REQUESTED** [review]
Just one question on the image tag

File: quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json (hunk 1 above)
Comment (erudinsky) [reply quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json:15]: @alex-frankel Thanks for taking a look at this. I was not sure what would be the correct placeholder by looking at [this page](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#parameters-file-placeholders) hence picked this one. What I want here is `latest` as default docker tag, but possible to change as input prior deploying the template to any from [here](https://hub.docker.com/_/sonarqube/tags). What would be then correct placeholder, pehaps `latest`?

Comment (erudinsky) [reply quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json]: Made it latest.

Comment (alex-frankel) [reply quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json]: Just to be clear, I don't know enough about this specific resource type to know if `latest` is an acceptable value. Did you check that deploying with `latest` as the parameter value deploys this as expected?

Comment (erudinsky) [reply quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/azuredeploy.parameters.json]: I made the `latest` as a default value for `sonarqubeDockerTag` parameter and tested deployment. Yes, with `latest` it worked!

**Pull request merged**